Client/Frontend/AuthenticationManager/AuthenticationManagerConstants.swift

```diff
@@ -77,6 +83,9 @@ struct AuthenticationStrings {
 
     static let loginsTouchReason =
         NSLocalizedString("Use your fingerprint to access Logins now.", tableName: "AuthenticationManager", comment: "Touch ID prompt subtitle when accessing logins")
+    
+    static let privateModeReason =
+        NSLocalizedString("Use your fingerprint to start browsing privately.", tableName: "AuthenticationManager", comment: "Touch ID prompt subtitle when entering private mode")
 
```

Comment:
@thebnich Does it make sense to keep following this string convention or move to what we do in `Strings.swift`?